### ISSUE TITLE
[BUG] Dashboards 2.x failed to build on Windows Agent

### ISSUE THREAD

**Issue by peterzhuamazon** [opensearch-project/OpenSearch-Dashboards#2554]
```
+ yarn build-platform --all-platforms --skip-os-packages --version-qualifier=
yarn run v1.22.19
$ node scripts/build --all-platforms --skip-os-packages --version-qualifier=
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating

 info [  global  ] Verifying environment meets requirements
   â”‚ succ Node.js version verified
   â”‚ succ âœ“ 0 sec

 info [  global  ] Cleaning artifacts from previous builds
   â”‚ debg Deleting patterns: [
   â”‚        'C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpr_kd6u__\\OpenSearch-Dashboards\\build',
   â”‚        'C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpr_kd6u__\\OpenSearch-Dashboards\\target',
   â”‚        'C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpr_kd6u__\\OpenSearch-Dashboards\\.node_binaries'
   â”‚      ]
   â”‚ debg Deleted 1 files/directories
   â”‚ succ âœ“ 0 sec

 info [  global  ] Downloading node.js builds for all platforms
   â”‚ debg Downloading shasum values for node version 14.20.0 from https://nodejs.org/dist/v14.20.0/SHASUMS256.txt
   â”‚ debg Attempting download of https://nodejs.org/dist/v14.20.0/node-v14.20.0-linux-x64.tar.gz 921680e244b813d6ffe06a4c22f87f05f39be635973c79486b2ded12a946cb37
   â”‚ debg Attempting download of https://nodejs.org/dist/v14.20.0/node-v14.20.0-darwin-x64.tar.gz 4471111f666f1d1ec549b002e0d416a7222c0fd416aa62c90b19ff3930b07dba
   â”‚ debg Attempting download of https://nodejs.org/dist/v14.20.0/node-v14.20.0-linux-arm64.tar.gz d2d15363a2f3a0446983d51a90af7942fe8b1dd4a7f16128dfe718b3bf56dc07
   â”‚ debg Attempting download of https://nodejs.org/dist/v14.20.0/win-x64/node.exe ac38e3cc6ba00d9209b1890da4e224e6ae819e681a25cc933770c253d40cab46
   â”‚ debg Downloaded https://nodejs.org/dist/v14.20.0/node-v14.20.0-darwin-x64.tar.gz and verified checksum
   â”‚ debg Downloaded https://nodejs.org/dist/v14.20.0/node-v14.20.0-linux-x64.tar.gz and verified checksum
   â”‚ debg Downloaded https://nodejs.org/dist/v14.20.0/node-v14.20.0-linux-arm64.tar.gz and verified checksum
   â”‚ debg Downloaded https://nodejs.org/dist/v14.20.0/win-x64/node.exe and verified checksum
   â”‚ succ âœ“ 1 sec

 info [  global  ] Extracting node.js builds for all platforms

   â”‚ succ âœ“ 17 sec

 info [  opensearch-dashboards  ] Copying source into platform-generic build directory

   â”‚ERROR failure 14 sec
   â”‚ERROR Error: EINVAL: invalid argument, utime 'C:\Users\ADMINI~1\AppData\Local\Temp\tmpr_kd6u__\OpenSearch-Dashboards\build\opensearch-dashboards\yarn.lock'
```

Running on Windows 2019 with nodejs 14.20.0 installed.
Able to run on Ubuntu with no issues.

Thanks.

**Comment by peterzhuamazon** [comment]
Hi @opensearch-project/opensearch-dashboards-core could you take a look at this?
Thanks.

**Comment by seanneumann** [comment]
@AMoo-Miki is looking at this and will respond by EOD Monday.

**Comment by peterzhuamazon** [comment]
@AMoo-Miki latest code in his fork is able to build windows on windows agent.

**Comment by kavilla** [comment]
@AMoo-Miki  can this be closed?

**Comment by AMoo-Miki** [comment]
Thanks Peter for confirming. This is fixed by #2601.
